axon-server-connector/src/test/java/org/axonframework/axonserver/connector/heartbeat/SchedulerTest.java

```diff
@@ -0,0 +1,70 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import org.junit.*;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for {@link Scheduler}
+ *
+ * @author Sara Pellegrini
+ * @since 4.2
```

Comment:
Nit: I believe we do not use the `since` tag for test classes. Not sure what convention would be best in this respect.

Reply:
I've removed this tag.